.phpunit.result.cache

```diff
@@ -0,0 +1 @@
LETE"";d:0.001;s:89:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "aura-PATCH"";d:0.001;s:88:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "aura-HEAD"";d:0.001;s:91:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "aura-OPTIONS"";d:0.001;s:93:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-GET"";d:0.001;s:94:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-POST"";d:0.001;s:93:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-PUT"";d:0.001;s:96:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-DELETE"";d:0.001;s:95:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-PATCH"";d:0.001;s:94:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-HEAD"";d:0.001;s:97:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "fast-route-OPTIONS"";d:0.001;s:90:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-GET"";d:0.002;s:91:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-POST"";d:0.002;s:90:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-PUT"";d:0.002;s:93:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-DELETE"";d:0.002;s:92:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-PATCH"";d:0.002;s:91:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-HEAD"";d:0.001;s:94:"MezzioTest\Router\IntegrationTest::testMatchWithAllHttpMethods with data set "laminas-OPTIONS"";d:0.001;s:99:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "aura-head"";d:0.001;s:102:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "aura-options"";d:0.001;s:105:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "fast-route-head"";d:0.001;s:108:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "fast-route-options"";d:0.001;s:102:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "laminas-head"";d:0.002;s:105:"MezzioTest\Router\IntegrationTest::testAllowedMethodsWhenOnlyPutMethodSet with data set "laminas-options"";d:0.002;s:119:"MezzioTest\Router\IntegrationTest::testWithOnlyRootPathRouteDefinedRoutingToSubPathsShouldDelegate with data set "aura"";d:0.001;s:125:"MezzioTest\Router\IntegrationTest::testWithOnlyRootPathRouteDefinedRoutingToSubPathsShouldDelegate with data set "fast-route"";d:0.001;s:122:"MezzioTest\Router\IntegrationTest::testWithOnlyRootPathRouteDefinedRoutingToSubPathsShouldDelegate with data set "laminas"";d:0.002;}}}
```

Comment:
I think this one accidentally slipped through. 😄

Reply:
That's why I shouldn't be coding after midnight :rofl: